config.json

```diff
@@ -335,401 +344,426 @@
+          "control_flow_loops",
           "math"
         ]
       },
       {
-        "slug": "resistor-color-duo",
-        "name": "Resistor Color Duo",
-        "uuid": "93271b2e-8ad9-4e7d-8732-5988140e8c00",
+        "slug": "clock",
+        "name": "Clock",
+        "uuid": "b868eeff-400c-4e46-8675-10d85c14b503",
         "practices": [],
         "prerequisites": [],
-        "difficulty": 1,
+        "difficulty": 4,
         "topics": [
-          "arrays",
-          "enums"
+          "control_flow_if_statements",
+          "preprocessor_x_macros_in_test",
+          "strings",
+          "text_formatting"
         ]
       },
       {
-        "slug": "resistor-color-trio",
-        "name": "Resistor Color Trio",
-        "uuid": "6cd7bb9c-2817-4c44-8bef-5f2916b2db2d",
+        "slug": "sieve",
+        "name": "Sieve",
+        "uuid": "fb2530a3-cbbf-4ba9-88d0-203faac03e96",
         "practices": [],
         "prerequisites": [],
-        "difficulty": 1,
+        "difficulty": 4,
         "topics": [
-          "arrays",
-          "enums",
-          "structs"
+          "control_flow_if_statements",
+          "math",
+          "memory_management"
         ]
       },
       {
-        "slug": "acronym",
-        "name": "Acronym",
-        "uuid": "1a8ba121-d2c4-4994-8c6b-610d8987346d",
+        "slug": "square-root",
+        "name": "Square Root",
+        "uuid": "0400f52d-48ae-4300-a72e-32e8a08514d0",
         "practices": [],
         "prerequisites": [],
-        "difficulty": 1,
+        "difficulty": 4,
         "topics": [
+          "bitwise_operations",
           "control_flow_loops",
-          "memory_management",
-          "strings"
+          "math"
         ]
       },
       {
-        "slug": "gigasecond",
-        "name": "Gigasecond",
-        "uuid": "000340f6-e30d-4d49-a255-016237d6fe60",
+        "slug": "reverse-string",
+        "name": "Reverse String",
+        "uuid": "a540d3d9-3c8b-46f5-a150-983308e02289",
         "practices": [],
         "prerequisites": [],
-        "difficulty": 3,
+        "difficulty": 4,
         "topics": [
-          "dates"
+          "loops",
+          "memory_management",
+          "strings"
         ]
       },
       {
-        "slug": "space-age",
-        "name": "Space Age",
-        "uuid": "fc03c7de-b0c6-4806-90bb-e9dda846e660",
+        "slug": "nucleotide-count",
+        "name": "Nucleotide Count",
+        "uuid": "38669990-e1e6-4486-a603-575a135d1f5c",
         "practices": [],
         "prerequisites": [],
-        "difficulty": 1,
+        "difficulty": 4,
         "topics": [
-          "functions"
+          "control_flow_loops_switch_if_statements",
+          "memory_management",
+          "strings",
+          "text_formatting"
         ]
       },
       {
-        "slug": "high-scores",
-        "name": "High Scores",
-        "uuid": "2c81f99a-991a-4914-aae7-d5bf72456119",
+        "slug": "protein-translation",
```

Comment:
Are there no topics for this exercise?
(Or perhaps I'm mis-reading the diff)

Reply:
Maybe fixing this is not in the scope of this PR?

Reply:
Good catch. While I'm here I'll add some in.